docs/source/tutorials/otx_cli/how_to_train/index.rst

```diff
@@ -0,0 +1,11 @@
+How to train, validate, export and optimize the model
+================================================================
+
+.. toctree::
+   :maxdepth: 1
+
+   classification
+   detection
+   instance_segmentation
+   semantic_segmentation
+   anomaly_classification
```

Comment:
```suggestion
   anomaly_detection
```

Reply:
Will we add one tutorial per anomaly task? (detection, classification, segmentation)